src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/PreviewTokenViewModel.cs

```diff
@@ -22,4 +22,6 @@ public record PreviewTokenViewModel
     public required DateTimeOffset Expiry { get; set; }
 
     public required DateTimeOffset? Updated { get; set; }
+    
+    public DateTimeOffset? Activates { get; set; }
```

Comment:
I think this can be 

```csharp
public required DateTimeOffset? Activates { get; set; }
```

or

```csharp
public required DateTimeOffset Activates { get; set; }
```

if you decide to make it non-nullable.